## The paper
### Genetic analysis of praziquantel response in schistosome parasites implicates a transient receptor potential channel
#### Le Clec'h W, Chevalier FD, Mattos ACA, Strickland A, Diaz R, McDew-White M, Rohr CM, Kinung'hi S, Allan F, Webster BL, Webster JP, Emery AM, Rollinson D, Djirmay AG, Al Mashikhi KM, Al Yafae S, Idris MA, Moné H, Mouahid G, LoVerde P, Marchant JS, Anderson TJC.
---
Mass drug administration with praziquantel (PZQ) monotherapy is considered the mainstay for control and elimination of the parasites causing schistosomiasis in humans. This drug shows imperfect cure rates in the field, and parasites showing reduced PZQ response can be selected in the laboratory, but the extent of resistance in Schistosoma mansoni populations is unknown. We examined the genetic basis of the variation in response in a PZQ-selected S. mansoni population (SmLE-PZQ-R) in which 35% of the parasitic worms survive high-dose PZQ (73 micrograms per milliliter) treatment. We used genome-wide association to map loci underlying PZQ response and identified a transient receptor potential (Sm.TRPMPZQ) channel (Smp_246790) within the major chromosome 3 peak that is activated by nanomolar concentrations of PZQ. The PZQ response showed recessive inheritance and marker-assisted selection of parasites at a single Sm.TRPMPZQ SNP that produced populations of PZQ-enriched resistant (PZQ-ER) and PZQ-enriched sensitive (PZQ-ES) parasites, exhibiting >377-fold difference in PZQ response. The PZQ-ER parasites survived treatment in rodents at higher frequencies compared with PZQ-ES, and resistant parasites exhibited 2.25-fold lower expression of Sm.TRPMPZQ relative to sensitive parasites. Specific chemical blockers of Sm.TRPMPZQ enhanced PZQ resistance, whereas Sm.TRPMPZQ activators increased sensitivity. We surveyed Sm.TRPMPZQ sequence variations in 259 parasites from different global sites and identified one nonsense mutation that resulted in a truncated protein with no PZQ binding site. Our results demonstrate that Sm.TRPMPZQ underlies variation in PZQ responses in S. mansoni and provides an approach for monitoring emerging PZQ-resistant alleles in schistosome elimination programs.

## Using WormBase ParaSite

In this paper, the researchers examined the genetic basis of the variation in response in a praziquantel PZQ-selected S. mansoni population (SmLE-PZQ-R) in which 35% of the parasitic worms survive high dose (73 μg/mL) PZQ treatment. 

They used genome-wide association and identified a transient receptor potential (Sm.TRPMPZQ) channel (Smp_246790) within the major chromosome 3 peak that is activated by nanomolar concentrations of PZQ.

We surveyed Smp_246790 sequence variations in 259 wild-type parasites from different global sites and identified one nonsense mutation that resulted in a truncated protein with no PZQ binding site using exome sequencing.

Let's try to follow the same steps that these researchers used to perform this part of their analysis using WormBase ParaSite.
---

### Assess the effects of variants the researchers identified when sequenced the Smp_246790 gene in 259 wild-type parasites.

WormBase ParaSite allows researchers to assess the effects of their variation datasets through the Variation Effect Predictor (VEP) tool.

The Variant Effect Predictor (VEP) determines the effect of your variants (SNPs, insertions, deletions, CNVs or structural variants) on genes, transcripts, and protein sequence, as well as regulatory regions through a simple point-and-click interface. Simply input the coordinates of your variants and the nucleotide changes to find out the:

- Genes and Transcripts affected by the variants.
- Location of the variants (e.g. upstream of a transcript, in coding sequence, in non-coding RNA, in regulatory regions).
- Consequence of your variants on the protein sequence (e.g. stop gained, missense, stop lost, frameshift).
- SIFT and PolyPhen-2 scores for changes to protein sequence.
... And more!

----

## Task 1: Use VEP to assess the effects of the variants our researchers sequenced

<details closed>
<summary>Click here to view the Smp_246790 gene variants in VCF format</summary>
SM_V9_3 2627949 . G GATA . . . <br>
SM_V9_3 2674963 . T C . . . <br>
SM_V9_3 2675058 . G A . . . <br>
SM_V9_3 2680940 . C A . . . <br>
SM_V9_3 2680944 . G T . . . <br>
SM_V9_3 2681001 . T G . . . <br>
SM_V9_3 2681047 . G T . . . <br>
SM_V9_3 2681100 . A T . . . <br>
SM_V9_3 2681130 . T C . . . <br>
SM_V9_3 2681142 . CAGG AAGA . . . <br>
SM_V9_3 2681186 . C A . . . <br>
SM_V9_3 2681187 . A T . . . <br>
SM_V9_3 2681190 . A T . . . <br>
SM_V9_3 2681197 . A G . . . <br>
SM_V9_3 2681229 . T G . . . <br>
SM_V9_3 2681289 . G A . . . <br>
SM_V9_3 2681305 . T C . . . <br>
SM_V9_3 2681306 . T C . . . <br>
SM_V9_3 2681352 . T C . . . <br>
SM_V9_3 2681476 . G A . . . <br>
SM_V9_3 2681485 . G A . . . <br>
SM_V9_3 2681491 . G A . . . <br>
SM_V9_3 2681503 . A G . . . <br>
SM_V9_3 2681534 . T A . . . <br>
SM_V9_3 2681560 . G A . . . <br>
SM_V9_3 2681578 . A G . . . <br>
SM_V9_3 2681604 . A T . . . <br>
SM_V9_3 2681610 . T C . . . <br>
SM_V9_3 2681652 . TCGT ACGA . . . <br>
SM_V9_3 2681666 . C A . . . <br>
SM_V9_3 2681678 . C T . . . <br>
SM_V9_3 2684386 . C T . . . <br>
SM_V9_3 2688007 . T C . . . <br>
SM_V9_3 2692611 . T C . . . <br>
SM_V9_3 2692622 . T C . . . <br>
SM_V9_3 2692658 . T C . . . <br>
SM_V9_3 2692675 . C A . . . <br>
SM_V9_3 2692681 . G T . . . <br>
SM_V9_3 2692687 . G T . . . <br>
SM_V9_3 2692693 . G T . . . <br>
SM_V9_3 2706056 . ACCGCC GCCACCGCT . . . <br>
SM_V9_3 2706073 . CACCACCACCAACAT TACCACCACCACCAACAA . . . <br>
SM_V9_3 2718303 . A G . . . <br>
SM_V9_3 2718312 . G A . . . <br>
SM_V9_3 2718326 . G T . . . <br>
SM_V9_3 2718365 . C T . . . <br>
SM_V9_3 2718367 . T C . . . <br>
SM_V9_3 2718374 . A G . . . <br>
SM_V9_3 2718383 . G T . . . <br>
SM_V9_3 2718383 . G A . . . <br>
SM_V9_3 2718449 . A T . . . <br>
SM_V9_3 2718486 . G A . . . <br>
SM_V9_3 2718509 . G T . . . <br>
SM_V9_3 2718526 . G A . . . <br>
SM_V9_3 2718564 . G C . . . <br>
SM_V9_3 2718567 . G T . . . <br>
SM_V9_3 2718594 . G C . . . <br>
SM_V9_3 2718603 . A T . . . <br>
SM_V9_3 2718637 . C A . . . <br>
SM_V9_3 2718644 . A G . . . <br>
SM_V9_3 2723071 . G T . . . <br>
SM_V9_3 2723103 . T A . . . <br>
SM_V9_3 2723137 . G A . . . <br>
SM_V9_3 2723138 . C T . . . <br>
SM_V9_3 2723141 . C T . . . <br>
SM_V9_3 2723153 . C G . . . <br>
SM_V9_3 2723154 . A G . . . <br>
SM_V9_3 2723187 . T C . . . <br>
SM_V9_3 2723196 . A G . . . <br>
SM_V9_3 2723298 . A T . . . <br>
SM_V9_3 2723331 . G T . . . <br>
SM_V9_3 2723349 . T C . . . <br>
SM_V9_3 2723363 . G A . . . <br>
SM_V9_3 2723368 . C T . . . <br>
SM_V9_3 2723376 . A T . . . <br>
SM_V9_3 2723390 . C A . . . <br>
SM_V9_3 2723408 . G A . . . <br>
SM_V9_3 2723414 . G A . . . <br>
SM_V9_3 2723533 . C A . . . <br>
SM_V9_3 2730952 . G T . . . <br>
SM_V9_3 2730963 . A T . . . <br>
SM_V9_3 2730981 . G T . . . <br>
SM_V9_3 2730989 . G A . . . <br>
SM_V9_3 2731010 . G T . . . <br>
SM_V9_3 2733021 . G T . . . <br>
SM_V9_3 2733032 . G T . . . <br>
SM_V9_3 2733033 . G C . . . <br>
SM_V9_3 2733033 . G A . . . <br>
SM_V9_3 2733055 . G T . . . <br>
SM_V9_3 2733056 . A G . . . <br>
SM_V9_3 2733071 . G A . . . <br>
SM_V9_3 2736834 . T C . . . <br>
SM_V9_3 2736852 . A G . . . <br>
SM_V9_3 2736854 . C T . . . <br>
SM_V9_3 2739034 . G T . . . <br>
SM_V9_3 2740214 . C T . . . <br>
SM_V9_3 2740329 . G T . . . <br>
SM_V9_3 2745612 . C T . . . <br>
SM_V9_3 2745615 . T A . . . <br>
SM_V9_3 2745725 . T C . . . <br>
SM_V9_3 2745757 . T A . . . <br>
SM_V9_3 2745779 . T C . . . <br>
SM_V9_3 2753367 . A G . . . <br>
SM_V9_3 2753399 . G T . . . <br>
SM_V9_3 2756226 . C A . . . <br>
SM_V9_3 2756238 . G T . . . <br>
SM_V9_3 2756249 . T C . . . <br>
SM_V9_3 2756264 . C T . . . <br>
SM_V9_3 2756266 . C A . . . <br>
SM_V9_3 2756276 . T C . . . <br>
SM_V9_3 2756321 . A G . . . <br>
SM_V9_3 2756324 . C T . . . <br>
SM_V9_3 2756340 . G T . . . <br>
SM_V9_3 2756343 . G A . . . <br>
SM_V9_3 2757916 . G T . . . <br>
SM_V9_3 2757921 . AAAA GAAG . . . <br>
SM_V9_3 2757939 . C A . . . <br>
SM_V9_3 2757999 . A T . . . <br>
SM_V9_3 2758008 . C T . . . <br>
SM_V9_3 2758011 . A G . . . <br>
SM_V9_3 2758077 . G A . . . <br>
SM_V9_3 2759653 . G A . . . <br>
SM_V9_3 2759657 . G T . . . <br>
SM_V9_3 2759675 . T C . . . <br>
SM_V9_3 2759693 . A G . . . <br>
SM_V9_3 2759726 . A G . . . <br>
SM_V9_3 2759733 . G T . . . <br>
SM_V9_3 2759741 . G A . . . <br>
SM_V9_3 2759753 . T G . . . <br>
SM_V9_3 2759765 . A G . . . <br>
SM_V9_3 2759768 . T A . . . <br>
SM_V9_3 2759781 . G T . . . <br>
SM_V9_3 2760337 . A G . . . <br>
SM_V9_3 2760344 . G A . . . <br>
SM_V9_3 2760379 . G T . . . <br>
SM_V9_3 2760400 . C A . . . <br>
SM_V9_3 2760408 . A G . . . <br>
SM_V9_3 2760417 . T C . . . <br>
SM_V9_3 2760419 . CACA TACC . . . <br>
SM_V9_3 2760437 . G A . . . <br>
SM_V9_3 2760471 . A G . . . <br>
SM_V9_3 2760483 . G A . . . <br>
SM_V9_3 2760497 . C T . . . <br>
SM_V9_3 2760499 . G T . . . <br>
SM_V9_3 2760511 . C G . . . <br>
SM_V9_3 2760514 . A C . . . <br>
SM_V9_3 2760533 . C T . . . <br>
SM_V9_3 2760534 . GCCACCA ACCACCG . . . <br>
SM_V9_3 2760555 . A AC . . . <br>
SM_V9_3 2760555 . A GC . . . <br>
SM_V9_3 2766369 . T C . . . <br>
SM_V9_3 2766442 . G A . . . <br>
SM_V9_3 2766449 . C T . . . <br>
SM_V9_3 2766531 . A G . . . <br>
SM_V9_3 2766613 . A G . . . <br>
SM_V9_3 2766651 . T C . . . <br>
SM_V9_3 2766696 . G A . . . <br>
SM_V9_3 2769090 . A G . . . <br>
SM_V9_3 2769148 . G T . . . <br>
SM_V9_3 2769188 . C T . . . <br>
SM_V9_3 2772552 . G T . . . <br>
SM_V9_3 2772567 . G A . . . <br>
SM_V9_3 2772582 . A G . . . <br>
SM_V9_3 2772588 . A G . . . <br>
SM_V9_3 2772594 . A G . . . <br>
SM_V9_3 2772604 . C A . . . <br>
SM_V9_3 2629023 . A C . . . <br>
SM_V9_3 2631820 . C T . . . <br>
SM_V9_3 2631849 . CATTTG TTTTTTA . . . <br>
SM_V9_3 2631862 . A T . . . <br>
SM_V9_3 2632680 . CTT TTC . . . <br>
SM_V9_3 2641704 . G A . . . <br>
SM_V9_3 2642036 . T C . . . <br>
SM_V9_3 2644055 . C CT . . . <br>
SM_V9_3 2644129 . G A . . . <br>
SM_V9_3 2646445 . G A . . . <br>
SM_V9_3 2646445 . G T . . . <br>
SM_V9_3 2647657 . A C . . . <br>
SM_V9_3 2648392 . T C . . . <br>
SM_V9_3 2648816 . T C . . . <br>
SM_V9_3 2654954 . C G . . . <br>
SM_V9_3 2654974 . C A . . . <br>
SM_V9_3 2654974 . C G . . . <br>
SM_V9_3 2664203 . C CA . . . <br>
SM_V9_3 2666719 . C T . . . <br>
SM_V9_3 2667585 . C CT . . . <br>
</details>

VCF is a text file format which is used to store information about a position in the genome. The format also has the ability
to contain genotype information on samples for each position and is widely used as the gold standard format for storing variation information. Learn more [here](https://samtools.github.io/hts-specs/VCFv4.2.pdf).

<details closed>
<summary>Solution</summary>
1. Go to WormBase ParaSite (https://parasite.wormbase.org/).<br><br>
2. Click ”Tools" at the top menu.<br><br>
3. Click "Variant Effect Predictor" on the "Processing your data" table.<br><br>
4. When you reach the VEP web interface, you will be presented with a form to enter your data and alter various options.<br><br>
5. First select the correct species for your data. Start typing into the box to get an autocomplete list of species. Make sure the species is set to "Schistosoma mansoni (PRJEA36577), (Tax ID 6183)".<br><br>
6. You can optionally choose a name for the data you upload - this can make it easier for you to identify jobs and files that you have uploaded to the VEP at a later point.
7. Paste all the lines with the variants in the VCF format from above into the "Either paste data:" dialog box.
8. Leave all extra options ("Identifiers","Additional annotations","Filtering options") to their default values. If you want to learn more about these extra options see [here](https://parasite.wormbase.org/info/Tools/vep.html).
9. Click "Run". 
10. Once you have clicked "Run", your input will be checked and submitted to the VEP as a job. All jobs associated with your session or account are shown in the "Recent Tickets" table. You may submit multiple jobs simultaneously.
</details>

----

#### Interpreting VEP results

The VEP presents a summary and a detailed results preview on its results page.

<ins>Statistics</ins>

Various statistics are listed in a table, including:

- Variants processed: any variants not parsed by the VEP are not included in this count
- Variants remaining: after filtering
- Novel / known variants: the number and percentage of novel variants vs existing variants in the input (see input page documentation)
- Number of overlapped genes, transcripts and regulatory features

<ins>Pie charts</ins>

Pie charts are shown detailing the proportion of consequence types called across all variants in the results. The colour scheme of the pie chart matches the colours used to draw variants on the Ensembl region in detail view.

![VEP pie chart](https://parasite.wormbase.org/img/vep_web_pie1.png)

<ins>Results preview table</ins>
The results table shows one row per transcript and variant. By default all of the columns are shown; to temporarily hide columns, click the blue "Show/hide columns" button and select or deselect the columns you wish to view. The columns you select will be recalled when viewing other jobs.

Hover over a column title to see a description. See the [Ensembl VEP documentation](http://www.ensembl.org/info/docs/tools/vep/vep_formats.html#output) for more details on each of the results columns.

To download what you see in the table, hover over the spreadsheet icon in the top right corner of the table.

Several columns have special features for the data they contain:

- Location: click the link to navigate to the region in detail view for the region surrounding this variant
- Gene, Feature and Existing Variation: click the link to bring up a summary view of the gene, transcript, regulatory feature or variation, from which you can navigate to the main Ensembl page for it
- Consequence: hover over the consequence name to see the Sequence Ontology definition.

<ins>Filtering results</ins>

You can apply any combination of filters to your results in order to identify interesting data.

To add a filter, simply select the column you wish to filter on, select an "operator", and input a value for the filter to compare to.

Certain columns when selected have special features:

- Location: for this column you may enter genomic coordinates in the format "chromosome:start-end".
- Allele, Feature type, Consequence and Biotype: for these columns, autocomplete will help you fill in the value when you start typing

<ins>Downloading results</ins>

The VEP allows you to download either your full or filtered results set in a choice of data formats.

----

## Task 2: Based on the VEP result, which variants you think will have the greater effects on the protein function?

<details>
<summary>Hint</summary>
- Use the "Filters" console and apply relevant filters to the "Consequence" and "Impact" columns.
- Use the "Show/Hide columns" button to show the "Protein position" and "Amino Acid" columns.
</details>

The researchers concluded: "We were able to sequence 36 of 41 exons of Sm.TRPMPZQ from 122 parasites (Table S3). We found 1 putative PZQ-R SNP in our reads supported by a very high read depth (Table S1, Fig. S7). This SNP (c.2708G>T on isoform 5, p.G903*) was found in a single Omani sample and resulted in a truncated protein predicted to result in loss-of-function, indicating that PZQ-R alleles are present in natural populations. However, this PZQ-R allele observed was rare and present in a heterozygous state so would not impact PZQ response (Fig. S8)." 

Was the variant they state in your list?



----
